README.md

```diff
@@ -261,6 +261,7 @@ Usage of oauth2_proxy:
   -client-secret string: the OAuth Client Secret
   -config string: path to config file
   -cookie-domain string: an optional cookie domain to force cookies to (ie: .yourcompany.com)
+  -cookie-path string: an optional cookie path to force cookies to (ie: .yourcompany.com/foo)
```

Comment:
Should the path example include the domain? The path is just meant to be `/foo` not `.yourcompany.com/foo` when set isn't it?

Reply:
Correct, the path value is just `/foo` without the domain. I considered a better example if used in the context of -cookie-domain, the parameter right above, but I guess makes sense to have just the example of the path in there.
Changed it to `/foo`.

Reply:
Maybe I'm being a little overly cautious, I just wondered if people might assume the value they should put is `<domain>/<uri>` based on that hint 🤷‍♂️  Thanks for updating